pandas/tests/frame/methods/test_shift.py

```diff
@@ -656,3 +656,73 @@ def test_shift_axis1_many_periods(self):
 
         shifted2 = df.shift(-6, axis=1, fill_value=None)
         tm.assert_frame_equal(shifted2, expected)
+
+    def test_shift_with_iterable_basic_functionality(self):
+        # GH#44424
```

Comment:
Can you add this comment to the start of all added tests

Reply:
👍